### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by harrington101** [xiaolin/react-image-gallery#153]
I'm trying to make my site more accessible to pass a WAVE Accessibility review but the image gallery is causing it to fail because the various buttons are not labeled. Can this component be updated to make use of aria-label on the bullets, play, full screen, left, and right buttons? Something like aria-label="slide 1", aria-label="slide 2", aria-label="next slide", etc.

**Comment by harrington101** [comment]
Great! I just sent it. #154

**Comment by xiaolin** [comment]
closing this as its released in v0.7.13
